### ISSUE TITLE
New Consoles Might Be Rotated Undesirably

### ISSUE THREAD

**Issue by Zytolg** [tgstation/tgstation#32802]
Issue reported from Round ID: 78838 (Sybil)

[Directions]: # (Include the Round ID from the Status panel or retrieve it from https://atlantaned.space/newSS13tools/round.php ! If you believe the issue to be caused by a testmerge [OOC tab -> Show Server Revision], report it in the pull request's comment section instead. Explain your issue in detail, including the steps to reproduce it.)

[For Admins]: # (Oddities induced by var-edits and other admin tools are not necessarily bugs. Verify that your issues occur under regular circumstances before reporting them.)

Sybil just got updated (yay!) however, it seems that some of the consoles in their quest for orientation have become confused and face walls among other things. Take Metastation's Medbay Crew Monitoring Console which now faces a wall! While checking over the some other areas of the station showed a majority of other consoles to be in the correct orientation on Metastation, it might be worth checking over again just in the odd even that another console decided to turn itself around. 
![sorry janice](https://user-images.githubusercontent.com/33048583/32879591-9d99d97c-ca78-11e7-97c9-56a6c2307d3e.JPG)

**Comment by SpaceManiac** [comment]
Somewhat relatedly, it seems the N logo screen sprites haven't been updated, observed on Pubby, Bagil, round 78840:
![image](https://user-images.githubusercontent.com/222630/32884026-dffc2f78-ca6d-11e7-8ede-69c237d72e4d.png)

**Comment by praisenarsie** [comment]
@shizcalev

**Comment by ShizCalev** [comment]
Yeah, that one console was a bit weird because I wasn't sure how to rotate it since it's shared by both sides. Any suggestions for the facing?

**Comment by ShizCalev** [comment]
Working on a new standby sprite since the one FTL has doesn't have the animation. Will have it done in a few.
